src/dispatch/incident/flows.py

```diff
@@ -547,11 +688,21 @@ def incident_create_flow(*, incident_id: int, checkpoint: str = None, db_session
             participant.individual.email, incident.id, db_session
         )
 
-    log.debug("Sent incident welcome and announcement notifications.")
+    event_service.log(
+        db_session=db_session,
+        source="Dispatch App",
+        description="Participants announced and welcome messages sent",
```

Comment:
Same as above, not sure these more general messages are actual events.

Reply:
Yeah, I was on the fence with this one.